framework/src/modules/token/schemas.ts

```diff
@@ -317,10 +341,14 @@ export const genesisTokenStoreSchema = {
 					address: {
 						dataType: 'bytes',
 						fieldNumber: 1,
+						minLength: 20,
+						maxLength: 20,
```

Comment:
Why don't we use `ADDRESS_LENGTH` here like other places in the schema using their own constants like `LOCAL_ID_LENGTH`?

Reply:
I updated this on later PR 🙏